### PR TITLE
Add constants in Common

### PR DESCRIPTION
**Pull request by tanmaymanolkar1**
Support change for
AzureAD/ad-accounts-for-android#1812

### CODE REVIEW

**Review by shoatman — APPROVED** [review]
lgtm

File: common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java [hunk 1]
```diff
@@ -1590,6 +1590,18 @@ public static String computeMaxHostBrokerProtocol() {
...
+        /**
+         * Intune CompanyPortal's ContentProvider String Authority constant for Clearing App policies.
+         */
+        public static final String AUTHORITY_COMPANY_PORTAL =
+                "content://com.microsoft.intune.omadm.authenticator";
+
+        /**
+         * Intune CompanyPortal's ContentProvider method constant for Clearing App policies.
+         */
+        public static final String METHOD_COMPANY_PORTAL =
```
Comment (rpdome) [inline common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java:1602]: nit: let's rename the var to be more specific.

**Review by rpdome — APPROVED** [review]

File: common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java (hunk 1 above)
Comment (iamgusain) [reply common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java:1602]: COMPANY_PORTAL_CONTENT_PROVIDER_METHOD_ON_GLOBAL_SIGNOUT

File: common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java [hunk 2]
```diff
@@ -1590,6 +1590,18 @@ public static String computeMaxHostBrokerProtocol() {
         public static final String AUTHORITY =
                 "content://com.microsoft.intune.shareduserlessdataclear/datacollection";
 
+        /**
+         * Intune CompanyPortal's ContentProvider String Authority constant for Clearing App policies.
+         */
+        public static final String AUTHORITY_COMPANY_PORTAL =
```
Comment (iamgusain) [inline common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java:1596]: COMPANY_PORTAL_CONTENT_PROVIDER_AUTHORITY

Comment (iamgusain) [reply common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java:1596]: nit: I think these two constants should go into separate class.. CompanyPortalContentProviderCall

**Review by iamgusain — CHANGES_REQUESTED** [review]

**Pull request closed without merging**